.github/workflows/ci.yml

```diff
@@ -1,6 +1,15 @@
 name: CI
 
-on: ['push', 'pull_request']
+on:
+  push:
+    branches:
+      - 'master'
+      - '0.next'
+  pull_request:
+    branches:
+      - '*'
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
I started to remove cron from my plugins
If there is no input after xx days, it will deactivate the whole CI pipeline and no further PRs would work anymore
It would require manual activation again, quite annoying.

Reply:
Ok, removed due to low commit rate.